packages/@react-spectrum/table/src/InsertionIndicator.tsx

```diff
@@ -0,0 +1,50 @@
+import {classNames} from '@react-spectrum/utils';
+import {ItemDropTarget} from '@react-types/shared';
+import React, {useRef} from 'react';
+import styles from './table.css';
+import {useTableContext} from './TableView';
+import {useVisuallyHidden} from '@react-aria/visually-hidden';
+
+interface InsertionIndicatorProps {
+  target: ItemDropTarget,
+  rowProps: any
+}
+
+export default function InsertionIndicator(props: InsertionIndicatorProps) {
+  let {dropState, dragAndDropHooks} = useTableContext();
+  const {target, rowProps} = props;
+
+  let ref = useRef();
+  let {dropIndicatorProps} = dragAndDropHooks.useDropIndicator(props, dropState, ref);
+  let {visuallyHiddenProps} = useVisuallyHidden();
+
+  let isDropTarget = dropState.isDropTarget(target);
+
+  if (!isDropTarget && dropIndicatorProps['aria-hidden']) {
+    return null;
+  }
+
+  let style = {
+    position: 'absolute',
+    top: rowProps.style.top,
+    width: rowProps.style.width
+  } as React.CSSProperties;
+
+  return (
+    <div style={style} role="row" aria-hidden={dropIndicatorProps['aria-hidden']}>
+      <div
+        role="gridcell"
+        aria-selected="false"
+        className={
+          classNames(
+            styles,
+            'react-spectrum-Table-InsertionIndicator',
+            {
+              'react-spectrum-Table-InsertionIndicator--dropTarget': isDropTarget
+            }
+        )}>
```

Comment:
List renders the visually hidden div conditionally, why is that missing here? again, not wrong, just curious

Reply:
Because of how ListView items were laid out, they were stacked on top of each other in terms of z-index. So there wasn't a way to show the full insertion indicator below an item without it being cut off by the row below. So there is a fake (isPresentationOnly) indicator that renders above the item below, and we don't want to actually double up on drop targets.

In tableview, these positioned absolutely, they don't have the same issue, and don't need to double up with a fake one.

This answer also applies to your question in https://github.com/adobe/react-spectrum/pull/4008#discussion_r1102086646. I think in listview if we used `position: absolute` for positioning the insertion indicators like here, we wouldn't need the fake ones.